protocols/dcutr/src/handler/direct.rs

```diff
@@ -40,8 +40,8 @@ pub struct Handler {
 }
 
 impl ConnectionHandler for Handler {
-    type InEvent = void::Void;
-    type OutEvent = Event;
+    type FromSwarm = void::Void;
```

Comment:
`ConnectionHandler`s receive messages from `NetworkBehaviour`s so this should be `FromBehaviour`.

Reply:
@thomaseizinger Done. Sorry, I had a confusion identifying the items. In the issue it was mentioned an `InEvent` to be renamed to `FromSwarm` [here](https://github.com/libp2p/rust-libp2p/issues/2854#issuecomment-1236575423), but I realized that it was part of the `NetworkBehaviour` trait (and not `ConnectionHandler`). 

As the issue is a bit old, I imagine that since this element has disappeared in favor of the `on_swarm_event` function which seems to fulfill the same criteria, correct?

Anyway it was well marked in the title and the `FromBehaviour`/`ToBehaviour` logic is totally coherent, so it's my fault :)